jest.config.js

```diff
@@ -56,6 +56,9 @@ module.exports = {
 		'test/helpers$': '<rootDir>/' + configPath + '/helpers.js',
 		jetpackConfig:
 			'<rootDir>/jetpack/tools/js-tools/jest/jest-jetpack-config.js',
+		// Workaround for Jest not having ESM support yet
+		// Reference: https://t.ly/9ap_
+		uuid: require.resolve( 'uuid' ),
```

Comment:
From reviewing the URL in this comment, my understanding is that we can remove this workaround when we [upgrade to `uuid@9`](https://github.com/uuidjs/uuid/issues/451#issuecomment-1206284480). Is that accurate? It would likely be beneficial to note this in the inline comment.

Reply:
Ah, true. I haven't tested this myself, but based on the comments that seems the case. I'll update the inline comment to reflect this. Thanks 🙇 !

Reply:
I tried using `uuid` version 9.x but I still had to use this workaround to address the issue. There might be something else producing the issue, especially as it's not happening in Gutenberg unit tests which I assume we use an older version `uuid` 🤔 .